src/scss/components/_newsDescription.scss

```diff
@@ -0,0 +1,49 @@
+.description__container{
+    padding: 60px 20px;
+}
+
+.description {
+    padding:0px;
+    @include breakpoint (lg) {
+        display: flex;
+    }
+
+    &__image {
+        z-index: 1;
+        position: relative;
+        padding-bottom: 53px;
+        @include breakpoint (lg) {
+            padding-right: 40px;
+            margin: 80px 20px 15px 0px;
+            height: 461px;
+            min-width: 461px;
+            min-height: 461px;   
+        }
+    }
+
+    &__title {
+        padding-bottom: 24px;
```

Comment:
Add `color: var(--black);` so that the text takes the right color according to the design.

Reply:
Done!